.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Deployment
+
+on:
+  release:
+    types: [published]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args:
+                      build -c ./outlookmail
+              env:
+                  REFRESH_URL: ${{ secrets.REFRESH_URL }}
+                  REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
+                  CLIENT_ID: ${{ secrets.CLIENT_ID }}
+                  CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+            - name: Ballerina Push
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args:
+                      push
+              env:
+                  BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_ACCESS_TOKEN }}
```

Comment:
There was a problem in the release of OneDrive not defining the working directory here. Please check with @LakshanSS if that is necessary here too.

Reply:
@sachinira Yes, We need to add the working directory there, I'll fix it.